### ISSUE TITLE
Issue of scrolling with SEO sitemap plugin

### ISSUE THREAD

**Issue by Laijiajia** [Geta/geta-optimizely-sitemaps#84]
We are using sitemaps plugin for CMS12. As seen in the picture, it is not possible to scroll the righthand pane and you thus can't access all information. 
![image](https://github.com/Geta/geta-optimizely-sitemaps/assets/42692471/1c3d85de-3d62-4094-b3b1-3bbe75d6337a)

**Comment by jaymarvels** [comment]
I believe this is due to the UI changes in the CMS as part of update 461: [https://world.optimizely.com/releases/optimizely---pdate-461](https://world.optimizely.com/releases/optimizely---pdate-461)

The display container for this plugin will probably need some CSS like this:

{
  height: calc(100vh - 40px);
  overflow: auto;
}

**Comment by dariusz-wozniak** [comment]
Yes, plus - if you have multiple sitemaps, it's impossible to scroll the list of sitemaps :(

**Comment by huseinatst** [comment]
Hi, is there any solution for it? I have multiple sitemaps and it's getting really impossible to use this plugin.
![geta_sitemap1](https://github.com/Geta/geta-optimizely-sitemaps/assets/138574983/1e7018e8-28a5-444c-8771-3d573fd3acf1)

and I have to scroll out to load the page, and add the sitemap :(
![geta_sitemap2](https://github.com/Geta/geta-optimizely-sitemaps/assets/138574983/3ef969e6-f122-4bf4-8022-8a0413228ddc)

have any one found the solution, please share it.

**Comment by huseinatst** [comment]
Hi Oleksa,

![image](https://github.com/Geta/geta-optimizely-sitemaps/assets/138574983/8dc8460c-7e40-434a-b0e3-199fa35ee923)
I don't see this file in entire solution, how should I make this changes?

**Comment by pooja-saboo-fele** [comment]
Is there any alternate way we can add scroll functionality to sitemaps?

**Comment by senderfele** [comment]
are there any updates on this? the issue still persists

**Comment by kaspars-ozols** [comment]
Scrolling should be fixed in this PR: https://github.com/Geta/geta-optimizely-sitemaps/pull/132
